common/traits/wc_race_traits.txt

```diff
@@ -718,6 +718,60 @@ creature_lich = {
 	random = no
 	customizer = no
 }
+creature_wickerman = {	# Throsic ghoul counterpart
```

Comment:
So, the wicker constructs shouldn't rot as the normal undead do. The rot event is `WCUND.150`.
I'm fine with both rephrasing it or blocking.

Reply:
Wickers won't rot for now.